src/__main__.py

```diff
@@ -1,23 +1,30 @@
-
 import sys
 import webbrowser
+import os
 
-from flask import Flask, render_template, request
+from flask import Flask, render_template, request, redirect, url_for, send_from_directory
 from gevent.pywsgi import WSGIServer
 from PyQt5.QtWidgets import QApplication, QGridLayout, QLabel, QPushButton, QWidget
+from werkzeug.utils import secure_filename
+from werkzeug.middleware.shared_data import SharedDataMiddleware
 
+UPLOAD_FOLDER = 'uploads'
```

Comment:
I think the uploads folder should be outside of the project directory to remove potential git issues and complications with packaging into executables.